docs/api/schemas.md

```diff
@@ -3416,14 +3649,35 @@ Parameter represents a set value for the scope.
 | `startup_script`        | string                                                  | false    |              |                                                                                                                                                            |
 | `vscode_port_proxy_uri` | string                                                  | false    |              |                                                                                                                                                            |
 
+## codersdk.WorkspaceAgentStatus
+
+```json
+"connecting"
+```
+
+### Properties
+
+| Name          | Type   | Required | Restrictions | Description |
+| ------------- | ------ | -------- | ------------ | ----------- |
+| `*anonymous*` | string | false    |              |             |
+
+#### Enumerated Values
+
+| Property      | Value          |
```

Comment:
This table might be confusing for users.

Reply:
Agreed, commented on another one before I saw this.